From 5ec97d37b4075e15d05fe90ebcd1fbb2d992fdb1 Mon Sep 17 00:00:00 2001
From: MkfsSion <myychina28759@gmail.com>
Date: Tue, 14 Aug 2018 09:04:23 +0800
Subject: [PATCH 9/9] camera: 1.0-legacy: Build with BOARD_VNDK_VERSION=current

* BOARD_VNDK_VERSION=current m -j16 camera.device@1.0-impl-legacy
* m -j16 camera.device@1.0-impl-legacy
* Test: Build successfully

Change-Id: I105105c86f20a11133b79fdf9c18cdff26283366
Signed-off-by: MkfsSion <myychina28759@gmail.com>
---
 camera/device/1.0-legacy/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/camera/device/1.0-legacy/Android.bp b/camera/device/1.0-legacy/Android.bp
index 6b71a74..e05538b 100644
--- a/camera/device/1.0-legacy/Android.bp
+++ b/camera/device/1.0-legacy/Android.bp
@@ -29,5 +29,6 @@ cc_library_shared {
     include_dirs: [
         "frameworks/native/include/media/openmax"
     ],
+    header_libs: ["media_plugin_headers"],
     export_include_dirs: ["."]
 }
-- 
2.17.1

